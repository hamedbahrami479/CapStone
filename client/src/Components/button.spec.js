//Button.test.js

import React from 'react';
import SearchButton from './button';
import {render, fireEvent} from "@testing-library/react";
import userEvent from "@testing-library/user-event";

describe("Test for Button Component", () => {
    it("snapshot button component", () => {
        const {asFragment} = render(<SearchButton/>);
        expect(asFragment(SearchButton)).toMatchSnapshot();
});

it("should render button component", () => {
const {getByText} = render(<SearchButton/>)
expect(getByText("Search")).toBeInTheDocument();
});
});