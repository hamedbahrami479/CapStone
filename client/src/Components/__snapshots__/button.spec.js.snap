// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Button Component snapshot button component 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <button>
      Search
    </button>
  </div>
</DocumentFragment>
`;
